app/views/TeamChannelsView.tsx

```diff
@@ -520,10 +520,10 @@ class TeamChannelsView extends React.Component<ITeamChannelsViewProps, ITeamChan
 			return <BackgroundContainer loading />;
 		}
 		if (isSearching && !search.length) {
-			return <BackgroundContainer text={searchText ? I18n.t('No_team_channels_found') : ''} />;
```

Comment:
The message `No_team_channels_found` is been used just here. Could you delete it from the jsons too?